aces/Loris
# Webpack integration

This PR migrates us to Webpack and removes Grunt. Because Webpack runs compiled code in a function scope we need to explicitly attach to `window` when we depend on something being in the global scope. This will allow us to maintain backwards compatibility while moving to a modularized structure. Eventually we should not rely on any globally scoped objects and instead `import` what we need directly into the file.

e.g.
```
import StaticDataTable from '../../jsx/StaticDataTable';
```

Here is a good article contrasting Grunt and Webpack:
https://advancedweb.hu/2016/02/02/the-first-steps-from-grunt-to-webpack/

TL;DR
- simpler configuration
- improves development workflow
- enables much needed [modules](https://strongloop.com/strongblog/an-introduction-to-javascript-es6-modules/) on the front-end

To test the new workflow:
- make sure you `npm install`
- run `npm run watch`
- edit a JS file that is included in `webpack.config.js`
